src/cbmc/cbmc_main.cpp

```diff
@@ -41,11 +41,9 @@ int wmain(int argc, const wchar_t **argv_wide)
 int main(int argc, const char **argv)
 {
 #endif
-  try
```

Comment:
Please squash this with the first commit that introduce the change that you undo here.